ALVRTrackingInterface/ALVRTrackingInterface.cs

```diff
@@ -230,78 +246,86 @@ private void Update()
         public class TrackingSensitivity
         {
             // Tracking Sensitivity Multipliers
-            public static float EyeLid = 1.1f;
+            public static float EyeLid = 1.0f;
             public static float EyeSquint = 1.0f;
             public static float EyeWiden = 1.0f;
             public static float BrowInnerUp = 1.0f;
             public static float BrowOuterUp = 1.0f;
             public static float BrowDown = 1.0f;
-            public static float CheekPuff = 1.4f;
-            public static float CheekSuck = 2.72f;
-            public static float CheekRaiser = 1.1f;
-            public static float JawOpen = 1.1f;
-            public static float MouthApeShape = 2.0f;
+            public static float CheekPuff = 1.0f;
+            public static float CheekSuck = 1.0f;
+            public static float CheekRaiser = 1.0f;
+            public static float JawOpen = 1.0f;
+            public static float MouthApeShape = 1.0f;
             public static float JawX = 1.0f;
             public static float JawForward = 1.0f;
-            public static float LipPucker = 1.21f;
+            public static float LipPucker = 1.0f;
             public static float MouthX = 1.0f;
-            public static float MouthSmile = 1.22f;
-            public static float MouthFrown = 1.1f;
-            public static float LipFunnelTop = 1.13f;
-            public static float LipFunnelBottom = 8.0f;  //VERY NOT SENSITIVE
+            public static float MouthSmile = 1.0f;
+            public static float MouthFrown = 1.0f;
+            public static float LipFunnelTop = 1.0f;
+            public static float LipFunnelBottom = 1.0f;  //VERY NOT SENSITIVE
             public static float LipSuckTop = 1.0f;
             public static float LipSuckBottom = 1.0f;
-            public static float ChinRaiserTop = 0.75f;
-            public static float ChinRaiserBottom = 0.7f;
-            public static float MouthLowerDown = 2.87f;
-            public static float MouthUpperUp = 1.75f;
-            public static float MouthDimpler = 4.3f;
-            public static float MouthStretch = 3.0f;
-            public static float MouthPress = 10f;        //VERY NOT SENSITIVE
-            public static float MouthTightener = 2.13f;
-            public static float NoseSneer = 3.16f;
+            public static float ChinRaiserTop = 1.0f;
+            public static float ChinRaiserBottom = 1.0f;
+            public static float MouthLowerDown = 1.0f;
+            public static float MouthUpperUp = 1.0f;
+            public static float MouthDimpler = 1.0f;
+            public static float MouthStretch = 1.0f;
+            public static float MouthPress = 1.0f;        //VERY NOT SENSITIVE
+            public static float MouthTightener = 1.0f;
+            public static float NoseSneer = 1.0f;
         }
```

Comment:
Is the scaling done in the UnifiedExpressions? If so may would need to have the scalers removed from the external module.

Reply:
There is a new Mutator class that will scale the values per-user with both user-definable values and a calibration sequence. These should be okay to leave but they may be redundant with the Mutator.